console/bumper_ci_service.ts

```diff
@@ -22,7 +22,7 @@ if (BumperService.is_for_pre_release) {
     {
       filename: "./src/cli/commands/make.ts",
       replaceTheRegex:
-        `import { Rhum } from "https://deno.land/x/rhum@v1.1.4/mod.ts`,
+        /import \{ Rhum \} from "https\:\/\/deno.land\/x\/rhum\@v[0-0]\.[0-9]\.[0-9]\/mod.ts"/g,
       replaceWith:
         `import { Rhum } from "https://deno.land/x/rhum@v{{ thisModulesLatestVersion }}/mod.ts"`,
```

Comment:
tbbh you could just do `replaceTheRegex: /rhum@v[...]/g, replaceWith: `rhum@v{{ ... }}`? or is there a reason to do the whole url

Reply:
i just did a quick and dirty implementation

Reply:
a26f53046daae47c6b

Reply:
i added tests:

https://github.com/drashland/rhum/pull/80/files#diff-83783ac33429890becf78098debd95619663ffb8e6f1b1906d733d5e0f6710c5